marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/mapping/getEntitiesForMapping.api

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
I think this should then be called "getMappableEntities", and I think it's fine for now if the desc references the MappingController class to see what's actually in the response here.